docker/env_file_app_ci

```diff
@@ -56,5 +56,5 @@ BROKER_URL=amqp://guest:guest@rabbitmq:5672
 
 
 # drf-recaptcha
-RECAPTCHA_SECRET_KEY_IO_PUBLIC="testkey"
-RECAPTCHA_SECRET_KEY_IO_LOCAL="testkey"
+ RECAPTCHA_SECRET_KEY_IO_PUBLIC="testkey"
```

Comment:
I guess this space is a leftover from a comment, could you remove it?

Reply:
yes, I made unrelated changes. I will cover them at once.